backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Purchase_Invoice_Details.sql

```diff
@@ -41,199 +41,201 @@ CREATE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Purchase_Invoice_Details
-       COALESCE(pc.IsHU, FALSE)                                                     AS IsHU,
-       MAX(il.line)                                                                 AS line,
-       -- ts: QnD: appending the invoice line description to the product name.
-       -- TODO: create a dedicated field for it etc
-       il.Description,
-       il.ProductDescription,
-       COALESCE(pt.name, p.name)                                                    AS Name,
-       COALESCE(
-               CASE
-                   WHEN LENGTH(att.Attributes) > 15
-                       THEN att.Attributes || E'\n'
-                       ELSE att.Attributes
-               END,
-               ''
-       )                                                                            AS Attributes,
-       SUM(il.QtyenteredTU)                                                         AS HUQty,
-       piip.name                                                                    AS HUName,
-       SUM(il.QtyInvoicedInPriceUOM)                                                AS qtyinvoicedinpriceuom,
-       SUM(CASE
-               WHEN il.QtyEntered > 0 THEN il.QtyEntered
-                                      ELSE 0
-           END)                                                                     AS shipped,
-       SUM(CASE
-               WHEN il.QtyEntered < 0 THEN il.QtyEntered * -1
-                                      ELSE 0
-           END)                                                                     AS retour,
-       il.PriceActual,
-       il.PriceEntered,
-       COALESCE(uomt.UOMSymbol, uom.UOMSymbol)                                      AS UOM,
-       COALESCE(puomt.UOMSymbol, puom.UOMSymbol)                                    AS PriceUOM,
-       puom.StdPrecision,
-       SUM(il.linenetamt)                                                           AS linenetamt,
-       t.rate,
-       bpg.IsPrintTax,
-       -- in case there is no C_BPartner_Product, fallback to the default ones
-       COALESCE(NULLIF(bpp.ProductNo, ''), p.value)                                 AS bp_product_no,
-       COALESCE(NULLIF(bpp.ProductName, ''), pt.Name, p.name)                       AS bp_product_name,
-       p.value                                                                      AS p_value,
-       p.description                                                                AS p_description,
-       i.description                                                                AS invoice_description,
-       c.cursymbol,
-       report.getPricePatternForJasper(i.m_pricelist_id)                            AS PricePattern
-FROM C_InvoiceLine il
-         INNER JOIN C_Invoice i ON il.C_Invoice_ID = i.C_Invoice_ID
-         INNER JOIN C_BPartner bp ON i.C_BPartner_ID = bp.C_BPartner_ID
-         LEFT OUTER JOIN C_BP_Group bpg ON bp.C_BP_Group_ID = bpg.C_BP_Group_ID
+SELECT *
```

Comment:
same; pls do not format; I do not know what you changed

Reply:
done, just added order by line